WordPress/src/main/java/org/wordpress/android/util/EventBusAdapter.kt

```diff
@@ -0,0 +1,17 @@
+package org.wordpress.android.util
+
+import de.greenrobot.event.EventBus
+import javax.inject.Inject
+
+/**
+ * Provides an interface for [de.greenrobot.eventbus.EventBus] which can be mocked and used in unit tests.
+ */
```

Comment:
Nice, thank you for exposing me to that annotation. While I always try to avoid making injected classes act **like** `@Singleton`, it makes sense for this class. Changed in 47daf596c13594025a0c773288caf35651d5daba.